src/analyses/variable-sensitivity/value_set_abstract_object.cpp

```diff
@@ -392,12 +401,24 @@ void value_set_abstract_objectt::output(
   else
   {
     out << "value-set-begin: ";
-    for(auto const &value : values)
+
+    std::vector<std::string> output_values;
+    for(const auto &value : values)
+    {
+      std::ostringstream ss;
+      value->output(ss, ai, ns);
+      output_values.emplace_back(ss.str());
+    }
+    std::sort(output_values.begin(), output_values.end(), by_length);
+
+    bool comma = false;
+    for(auto const &value : output_values)
```

Comment:
:pick: There are existing unit tested functions for joining a collection of strings together with a delimiter and outputting the result to a stream. These are called `join_strings` and they can be found in `src/util/string_utils.h`. It is preferable to reuse the existing functions so that we don't have duplicate versions of the same functionality to maintain and because the existing functions are unit tested.

Reply:
Change made.